client/src/components/AuthButtons/GithubAuthButton.jsx

```diff
@@ -34,7 +34,8 @@ export default function GithubAuthButton() {
           })
           // .then((res) => console.log(res))
           .catch((err) => console.log(err));
-        navigate("/u/dashboard/account");
+        await auth.currentUser.reload()
```

Comment:
What are we using this for? @Ajen07

Reply:
We use this because when we signup with github the current firebase user's email is not verified , after verifying it in backend current is reloaded to fetch the updated user